modules/openapi-generator/src/main/resources/cpp-qt5-client/api-body.mustache

```diff
@@ -9,15 +9,15 @@
 namespace {{this}} {
 {{/cppNamespaceDeclarations}}
 
-{{classname}}::{{classname}}() {
+{{classname}}::{{classname}}() : basePath("{{{basePathWithoutHost}}}"){
```

Comment:
Whats the default value of host?

Reply:
Yes, I need to add this.
I thought of making these members private but it might break some things.

Reply:
Should the user really be able to change the base path and host after initializing the api?

Reply:
So private members would be better i think

Reply:
ok. then lets add a setter